examples/plot_alignment_methods_benchmark.py

```diff
@@ -0,0 +1,148 @@
+source_test = df[df.subject == 'sub-01'][df.acquisition == 'pa'].path.values
+target_test = df[df.subject == 'sub-02'][df.acquisition == 'pa'].path.values
+
+###############################################################################
+# Choose the number of regions for local alignment
+# ---------------------------------------------------------------------------
+# First, as we will proceed to local alignment we choose a suitable number of \
+# regions so that each of them is approximately 200 voxels wide. Then our \
+# estimator will first make a functional clustering of voxels based on train \
+# data to divide them into meaningful regions.
+#
+
+import numpy as np
+n_voxels = roi_masker.mask_img_.get_data().sum()
+print("The chosen region of interest contains {} voxels".format(n_voxels))
+n_pieces = int(np.round(n_voxels / 200))
+print("We will cluster them in {} regions".format(n_pieces))
+
+###############################################################################
+# Define the estimators, fit them and do a prediction
+# ---------------------------------------------------------------------------
+# On each region, we search for a transformation R that is either :
+#   *  orthogonal, i.e. R orthogonal, scaling sc s.t. ||sc RX - Y ||^2 is minimized
+#   *  a ridge regression : ||XR - Y||^2 + alpha *||R||^2 with a L2 penalization
+#      on the norm of R.
+#   *  the optimal transport plan, which yields the minimal transport cost
+#       while respecting the mass conservation constraints. Calculated with
+#       entropic regularization.
+#   *  we also include identity (no alignment) as a baseline.
+# Then for each method we define the estimator fit it, predict the new image and plot
+# its correlation with the real signal.
+#
+
+from fmralign.pairwise_alignment import PairwiseAlignment
+from fmralign._utils import voxelwise_correlation
+methods = ['identity', 'scaled_orthogonal', 'ridge_cv', 'optimal_transport']
+
+for method in methods:
+    alignment_estimator = PairwiseAlignment(
+        alignment_method=method, n_pieces=n_pieces, mask=roi_masker)
+    alignment_estimator.fit(source_train, target_train)
+    target_pred = alignment_estimator.transform(source_test)
+    aligned_score = voxelwise_correlation(target_test, target_pred, roi_masker)
+    title = "Correlation of prediction after {} alignment".format(method)
+    display = plotting.plot_stat_map(aligned_score, display_mode="z",
+                                     cut_coords=[-15, -5], vmax=1, title=title)
+
+#############################################################################
+# We can observe that all alignment methods perform better than identity \
+# (no alignment). Ridge and Optimal Transport perform better than Scaled \
+# Orthogonal alignment. Usually Ridge yields best scores for this kind of \
+# metrics but for real world problem we suspect it destroys signal specificity, \
+# and yiels very smooth predictions. \ Our recommandation is to use scaled \
```

Comment:
I dislike issuing such comments without proper arguments. I would soberly conclude that Ridge outperforms.
If you want to  criticize the ridge result, be specific.